### ISSUE TITLE
How to step into linear_attention.py/forward function ?

### ISSUE THREAD

**Issue by Yogurt928** [idiap/fast-transformers#57]
Thanks for author's sharing.
After installation, I can run the quickstart sample code successfully.
To better understand the framework, I want to see the data change under debug mode.
But under the debug mode, the project will not stop in the breakpoint I set in linear_attention.py/forward function

+-----------------------------------------------------------------------------+
(pytorch36) xxx:~/Code/fast-transformers/tests$ python -m pdb quickstart.py
> /Code/fast-transformers/tests/quickstart.py(1)<module>()
-> import torch
**(Pdb) b fast_transformers/attention/linear_attention.py:56**
Breakpoint 1 at /.local/lib/python3.6/site-packages/fast_transformers/attention/linear_attention.py:56
(Pdb) c
Softmax:  456.4417724609375 ms
Linear:  259.7819519042969 ms
The program finished and will be restarted

I also tried vscode in local windows machine to remote debug, but it can not step into linear_attention.py/forward() too.

Any suggestion about it?

Thanks!

**Comment by angeloskath** [comment]
Hi,

The quick start code does not use linear attention. You can set linear attention if you change `attention_type="full"` to `attention_type="linear"`.

I am closing the issue but feel free to reopen it if the above does not solve your problem.

Happy holidays,
Angelos
